tests/annotator/annotator.cpp

```diff
@@ -1957,11 +2009,11 @@ TEST(Annotator, hashUpdatedWithAllAutomaticIds)
     auto model = parser->parseModel(modelStringUniqueIds);
     auto annotator = libcellml::Annotator::create();
 
-    annotator->buildModelIndex(model);
-    EXPECT_TRUE(annotator->isBuilt());
+    annotator->setModel(model);
+    EXPECT_TRUE(annotator->hasModel());
 
     annotator->assignAllIds();
-    EXPECT_TRUE(annotator->isBuilt());
+    EXPECT_TRUE(annotator->hasModel());
```

Comment:
I understand the function names when `hasModel` is testing whether or not you have a model.  But I don't like it in this situation where it's actually testing that the stored model is up to date with the stored index.  Can we name it something else?

Reply:
Is this comment still relevant? @kerimoyle? @hsorby?